### PR TITLE
[ES-127] Sign-in-with-esignet story with different buttons added

### PR DESCRIPTION
**Pull request by YaswanthNagarjuna**

### CODE REVIEW

**Review by anshulv1401 — APPROVED** [review]

**Review by gsasikumar — APPROVED** [review]

File: storybook-example/stories/SignInWithEsignet.stories.ts [hunk 1]
```diff
@@ -2,6 +2,35 @@ import { Meta, StoryObj } from "@storybook/web-components";
...
+
+const buttonConfig = {
+  type: "standard",
+  theme: "outline",
+  shape: "sharp_edges",
+  labelText: "Sign in with e-Signet",
+};
+
+const oidcConfigType =
+  '{\r\n  authorizeUri: "string",\r\n  redirect_uri: "string",\r\n  client_id: "string",\r\n  scope: "string",\r\n  nonce: "string",\r\n  state: "string",\r\n  acr_values: "string",\r\n  claims_locales: "string",\r\n  ui_locales: "string",\r\n  display: "page" | "popup" | "touch" | "wap",\r\n  max_age: "number",\r\n  prompt: "none" | "login" | "consent" | "select_account",\r\n}';
```
Comment (gsasikumar) [inline storybook-example/stories/SignInWithEsignet.stories.ts:29]: why do you need /r/n?

**Review by gsasikumar — CHANGES_REQUESTED** [review]

File: storybook-example/stories/SignInWithEsignet.stories.ts (hunk 1 above)
Comment (YaswanthNagarjuna) [reply storybook-example/stories/SignInWithEsignet.stories.ts:29]: Added /r/n for the breaking the line and starting the new line

Comment (YaswanthNagarjuna) [reply storybook-example/stories/SignInWithEsignet.stories.ts:29]: If we add /r/n then it looks like proper json format
Screenshot 2023-07-13 113653

**Review by gsasikumar — APPROVED** [review]

**Review by gsasikumar — APPROVED** [review]

**Pull request merged**